.editorconfig

```diff
@@ -7,7 +7,7 @@ indent_style = space
 indent_size = 4
 
 [package.json]
-indent_size = 2
+indent_size = 4
```

Comment:
I just meant for the code, package.json + other existing configs could stay at 2 spaces for simplicity. Many programs that generate them automatically only use 2 spaces (like `npm install` for example). Best leave it like that perhaps